lib/cli.ex

```diff
@@ -36,6 +48,7 @@ defmodule Onigumo.CLI do
     COMPONENT\tOnigumo component to run, available: #{components}
 
     OPTIONS:
+    -h, --help\t\tPrint this help
```

Comment:
Why two tabs? We should not assume tab size. Maybe we shouldn’t use tabs at all. For now, let’s use `\t` as in `-C`.

Created #265.

```suggestion
    -h, --help\tPrint this help
```

Reply:
Why to remove double `TAB`?